queue_processors/queue_processor/queueproc_utils/tests/test_script_utils.py

```diff
@@ -0,0 +1,56 @@
+# ############################################################################### #
+# Autoreduction Repository : https://github.com/ISISScientificComputing/autoreduce
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI
+# SPDX - License - Identifier: GPL-3.0-or-later
+# ############################################################################### #
+import os
+from unittest.mock import Mock, patch
+from queue_processors.queue_processor.queueproc_utils.script_utils import import_module
+from queue_processors.queue_processor.queueproc_utils.tests.module_to_import import TEST_DICTIONARY
+
+
+def assert_raises(exc, func, args):
```

Comment:
Can we not just use:
```python
with pytest.raises(ExceptionClass):
```
And remove custom assert.

As a side note we have a mixture of unittest testcase classes and just test modules written the pytest way. We should probably decide on one way and move towards making them all the same in the future. Not a big deal for now though.

Reply:
Thing is both unittest and pytest do some things better. As long as they can be used without interfering with each other I think a mix is OK. Though I lean towards pytest more and more